app/routes/document-response.js

```diff
@@ -0,0 +1,141 @@
+const categories = [
```

Comment:
Will be better to move these out into a constants module.

Reply:
Sorted